oxide-auth-async/src/endpoint/mod.rs

```diff
@@ -49,12 +54,22 @@ pub trait Endpoint<Request: WebRequest> {
     /// Wrap an error in the request/response types.
     fn web_error(&mut self, err: Request::Error) -> Self::Error;
 
-    /*
     /// Get the central extension instance this endpoint.
     ///
     /// Returning `None` is the default implementation and acts as simply providing any extensions.
     fn extension(&mut self) -> Option<&mut dyn Extension> {
         None
     }
-    */
+}
+
+pub trait Extension {
+    /// The handler for authorization code extensions.
```

Comment:
Please add a `FIXME` to things that are commented out and are somewhat central to the trait or would be a breaking change to introduce.